packages/core/src/index.ts

```diff
@@ -89,8 +89,12 @@ export abstract class BaseGraphQLServer<TContext> {
       'schema' in options
         ? options.schema
         : makeExecutableSchema({
-            typeDefs: options.typeDefs,
-            resolvers: options.resolvers,
+            typeDefs: [options.typeDefs, 'scalar File', 'scalar Blob'],
```

Comment:
we should bring that `enableUploads` option back so that if in future someone wants to move away it is more explicit how these types were getting added.